hw_13/tests/test_account_page.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+
+@pytest.mark.regression
+def test_edit_contact_info_visible(open_login_link, env):
+    login_link = open_login_link
+    email = env.email
+    password = env.password
+    account_page = login_link.account(email, password)
+    assert account_page.is_contact_edit_visible() is True
+
+
+@pytest.mark.regression
+def test_welcome_message_visible(open_login_link, env):
+    login_link = open_login_link
+    email = env.email
+    password = env.password
+    account_page = login_link.account(email, password)
```

Comment:
same here, you don't need variables. Variable is required if we use it more than 1 time. Also, you can create separate fixture login, to avoid code duplication in your tests and hide email and password from tests